scripts/apothecary/formulas/openSSL/buildwin.cmd

```diff
@@ -0,0 +1,46 @@
+@echo off
```

Comment:
For consistency, I would prefer, if possible, a shell file instead. What do you think?

And can't we use lib from official OpenSSL distribution?